src/reporting/clients/IExtractionClient.ts

```diff
@@ -59,4 +59,33 @@ export interface IExtractionClient {
     accessToken: AccessToken,
     jobId: string
   ): Promise<ExtractionStatus>;
+
+  /**
+   * Gets Extractions of an IModel. This method returns the full list of Extractions.
+   * @param {string} iModelId Unique Identifier of the target IModel.
```

Comment:
Documentation is inconsistent with the methods above.
```suggestion
   * @param {string} iModelId The iModel Id.
```